main.go

```diff
@@ -1,32 +1,69 @@
 package main
 
-import (
+import(
 	"fmt"
 	"log"
 	"net/http"
-
 	"github.com/EaseApp/web-backend/src/db"
-
 	"github.com/codegangsta/negroni"
 	"github.com/gorilla/mux"
+	user "github.com/EaseApp/web-backend/src/app/controllers/user"
+	application "github.com/EaseApp/web-backend/src/app/controllers/application"
+	dao "github.com/EaseApp/web-backend/src/app/dao"
 )
 
+
+
+func NewStaticUserHandler(w http.ResponseWriter, req *http.Request){
+	_ = user.InsertStaticUser()
+	http.Redirect(w, req, "/users", http.StatusFound)
+}
+
+func DBCountHandler(w http.ResponseWriter, req *http.Request){
+	vars := mux.Vars(req)
+	db := vars["db"]
+
+	// Move RecordCount out of user DAO and into generic DAO
+	fmt.Fprintf(w, "Db (%v) has (%v) objects.", db, dao.RecordCount(db))
+}
+
 func main() {
 	log.Println("Starting server...")
 
-	router := mux.NewRouter()
-	router.HandleFunc("/", func(w http.ResponseWriter, req *http.Request) {
-		fmt.Fprintf(w, "Welcome to the home page!")
-	})
-
 	// TODO: Use command line flag credentials.
 	client, err := db.NewClient("localhost:28015")
 	if err != nil {
 		log.Fatal("Couldn't initialize database: ", err.Error())
 	}
+	db.CreateEaseDb(client)
+	db.CreateUserTable(client)
+	db.CreateDbTable(client)
```

Comment:
The dbs and tables aren't overwritten by default, you just get an error and it's logged. I could see adding a check being a good val-cap